client/src/pages/ResetPasswordRequest.jsx

```diff
@@ -0,0 +1,125 @@
+import { Formik, Form, Field } from 'formik';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { useState } from 'react';
+import { usePageError } from '../hooks/usePageError.js';
+import { authService } from '../services/authService.js';
+
+function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+export const ResetPasswordRequest = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const [error, setError] = usePageError('');
```

Comment:
The `usePageError` hook is used incorrectly here. It should return a function to set the error, not the error itself. Consider using `const setError = usePageError('');` instead.